syntax = "proto3";

enum DeviceType
{
	DeviceAIS = 0;
	DeviceCCTV = 1;
	DeviceVHF = 2;
	DeviceRadar = 3;
	DeviceTrack = 4;
	DeviceMeteo = 5;
	DeviceSQL = 6;
	DeviceServer = 7;
}
message Device
{
    DeviceType type = 1;//设备类型
    bool status = 3;
}

message DeviceDetail
{
    string DeviceName = 1;
    string MonitorName = 2;
    bool status = 3;
    int64 time = 4;
}